.buildkite/scripts/steps/integration_tests_tf.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-source .buildkite/scripts/common2.sh
-
+source .buildkite/scripts/common-integration.sh
 source .buildkite/scripts/steps/ess.sh
+source .buildkite/scripts/steps/stable_ess_version.sh
```

Comment:
What's the reason for creating a step called `stable_ess_version` instead of adding the function to the `common-integration.sh` script itself?

Reply:
That's also an option

Reply:
Moved to common-integration.sh